src/styles/card.scss

```diff
@@ -0,0 +1,73 @@
+.card {
+  width: 198px;
+  height: 406px;
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+
+  &__image {
+    padding: 32px 19px 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 150%;
+    color: $card-title-color;
+    padding-bottom: 4px;
+    padding-left: 16px;
+    padding-right: 16px;
+    max-width: 198px;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), avoid setting `width` or `max-width` with the same value as the parent element. Since `.card` has a width of 198px, it's unnecessary to set the same `max-width` for its child elements. Consider removing this property if it doesn't affect the layout.